docs/events/tampere-20220917/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+
+    <title>ClojureBridge - Tampere 17.9.2022</title>
+
+    <meta charset="utf-8">
+    <meta content="IE=edge" http-equiv="X-UA-Compatible">
+    <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
+
+    <!-- General -->
+    <meta name="language" content="fi">
+    <meta name="keywords"
+          content="clojure, clojurebridge, programming, training, gender equality, finland, ohjelmointi, opiskelu, tasaarvo, suomi, helsinki">
+    <meta name="description" content="ClojureBridge-koulutus Helsingissä">
+    <meta name="subject" content="ClojureBridge Finland yhteisö">
+    <meta name="robots" content="index,follow">
+    <meta name="googlebot" content="index,follow">
+    <meta name="google" content="nositelinkssearchbox">
+    <meta name="abstract" content="ClojureBridge-koulutus Helsingissä">
+    <meta name="topic" content="ClojureBridge Finland yhteisö">
+    <meta name="summary" content="ClojureBridge Finland yhteisö">
+    <meta name="reply-to" content="clojurebridge@metosin.fi">
+    <meta name="owner" content="ClojureBridge Finland yhteisö">
+    <meta name="url" content="https://clojurebridge.fi/events/helsinki-20190511/">
+    <meta name="identifier-URL" content="https://clojurebridge.fi/events/helsinki-20190511/">
+
+    <!-- FB: -->
+    <meta property="og:url" content="https://clojurebridge.fi/events/helsinki-20190511/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="ClojureBridge-koulutus Helsingissä">
+    <meta property="og:image" content="http://clojurebridge.fi/img/cljbridge-zorcam.jpg">
+    <meta property="og:description" content="Ohjelmoinnin opetusta naisille Helsingissä">
+    <meta property="og:site_name" content="ClojureBridge Finland">
+    <meta property="og:locale" content="fi_FI">
+    <meta property="article:author" content="clojurebridge@metosin.fi">
+
+    <!-- Twitter: -->
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@ClojureBridgeFI">
+    <meta name="twitter:creator" content="@jarppe">
+    <meta name="twitter:url" content="https://clojurebridge.fi/events/helsinki-20190511/">
+    <meta name="twitter:title" content="ClojureBridge-koulutus Helsingissä">
+    <meta name="twitter:description" content="Ohjelmoinnin opetusta naisille Helsingissä">
+    <meta name="twitter:image" content="https://clojurebridge.fi/img/cljbridge-zorcam.jpg">
```

Comment:
`meta` tags refer to Helsinki instead of Tampere

Reply:
Thanks! Found some other typos as well, will fix asap! 👍